spec/services/solidus_bolt/transaction/capture_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+# rubocop:disable RSpec/MultipleMemoizedHelpers
```

Comment:
Usually we are disabling this cop globally:

https://github.com/nebulab/solidus_square/blob/main/.rubocop.yml#L10

Reply:
Added a new commit to disable globally